### ISSUE TITLE
Mocked object doesn't process mixin class getter/setter

### ISSUE THREAD

**Issue by domino2** [drashland/rhum#145]
## Summary

The getter and setter defined in the mixin class is not processed.
The mock see that property as undefined.

## Steps To Reproduce The Bug

```
import { Rhum } from "https://deno.land/x/rhum@v1.1.14/mod.ts";
import { assertInstanceOf } from "https://deno.land/std@0.133.0/testing/asserts.ts";

export interface IGame {}

export class NullGame implements IGame {}

function WithGameProperty() {
  return <T extends new (...args: any[]) => any>(GameProperty: T) => {
    return class extends GameProperty {
      private game: IGame = new NullGame();

      public set Game(game: IGame) {
        this.game = game;
      }

      public get Game() {
        return this.game;
      }
    };
  };
}

const PlayersEngine = WithGameProperty()(
  class PlayersEngine {
    onNextGenerationReady(listener: () => void): void {
      // add listener to react on an event
    }

    prepareNextGeneration(): void {
      // do some stuff and trigger an event
    }
  },
);

const pe = new PlayersEngine();
const mpe = Rhum.mock(PlayersEngine).create();

assertInstanceOf(pe.Game, NullGame);
assertInstanceOf(mpe.Game, NullGame);
```

## Expected Behavior

It will pass last line in the code above without an error.

**Comment by ebebbington** [comment]
Hello @domino2, I’m assuming you’re the one who posted on SO

Apologies that you’re still having issues with this, we’ll look into it :)

**Comment by domino2** [comment]
Yeah ... this is another issue actually (just related) :)
I'm looking forward.

**Comment by ebebbington** [comment]
thanks again :) this does seem to be working as expected though

`Game` is a getter/setter, what i think you want to be checking is `game` as thats the property with a type of `NullGame`

Though there will be a problem with the type hinting on this, `mock.game` will throw a ts error, so i will get back to you on passing a generic to the mock call

**Comment by ebebbington** [comment]
Oh i see your problem actally, `game` is initiased, so `pa.Game instanceof NullGame` should be true

**Comment by ebebbington** [comment]
@domino2 i've found a fix, i will release a new version soon

**Comment by crookse** [comment]
hey @domino2 ! i just released v1.1.15 that includes the fix @ebebbington made for this issue. you should be able to bump your rhum version and be good to go:

```ts
import { Rhum } from "https://deno.land/x/rhum@v1.1.15/mod.ts";
// or export { Rhum } from "https://deno.land/x/rhum@v1.1.15/mod.ts";
```

let us know if you have any issues. thanks !
